Makefile

```
@@ -3,27 +3,30 @@ calling_from_make:
 
 UNAME := $(shell uname)
 
-CFLAGS ?= -Wall -Werror -Wno-unused-parameter -pedantic -std=c99 -O2
+CFLAGS ?= -Wall -Werror -Wextra -Wno-unused-parameter -pedantic -O2 -fPIC
 
 ifeq ($(UNAME), Darwin)
-	TARGET_CFLAGS ?= -fPIC -undefined dynamic_lookup -dynamiclib -Wextra
+	TARGET_CFLAGS ?= -undefined dynamic_lookup -dynamiclib -Wextra
+else
+	TARGET_CFLAGS ?= -shared
+
+# c_src/spawner.c fails to compile on NetBSD and Darwin with -D_POSIX_C_SOURCE=200809L
+# Meanwhile *-linux-musl needs it
+ifneq (${UNAME}, NetBSD)
+		CFLAGS += -D_POSIX_C_SOURCE=200809L
```

```suggestion
	TARGET_CFLAGS ?= -undefined dynamic_lookup -dynamiclib -Wextra
else ifeq ($(UNAME), NetBSD)
	TARGET_CFLAGS ?= -shared
else
	TARGET_CFLAGS ?= -shared
	# c_src/spawner.c fails to compile on NetBSD and Darwin with -D_POSIX_C_SOURCE=200809L
	# Meanwhile *-linux-musl needs it
	CFLAGS += -D_POSIX_C_SOURCE=200809L
```

Nit. Just trying to reduce nested and negative if statements.

---

Right, applied a similar change together with linking to http://gnats.netbsd.org/cgi-bin/query-pr-single.pl?number=57871 for NetBSD's issue.